src/octopeer-github/content/Types/ElementID.ts

```diff
@@ -4,48 +4,57 @@
  */
 class ElementID {
 
+    /**
+     * The 'Getters in this list are not really getters,
+     * they are the closest thing TypeScript has to offer as
+     * public static final fields.
+     */
     /** Instantiated ElementID for the MergePullRequestButton. */
-    public static get MERGEPRREQUEST() { return new ElementID(101); }
+    public static get MERGE_PULLREQUEST() { return new ElementID(101); }
```

Comment:
Is there a reason why `PULLREQUEST` is fully written here, and things like `PR_CREATOR` have `PR` as abbreviation for it? Why not also use `PR` here?